MidiFile/MidiFile.h

```diff
@@ -179,12 +175,7 @@ class MidiFile
 
 	uint8_t* read_file(const std::string& file_name, unsigned int& file_size)
 	{
-#ifdef _WIN32

```

Comment:
I could not be bothered with wrapping all the sprintf calls like this... if it needs cross platform support, I'd rather write a header that defines these "_s" functions on other OSes.